apps/frontend/stores/current-repository.ts

```diff
@@ -22,9 +22,10 @@ export const useCurrentRepository = defineStore('currentRepository', () => {
   });
 
   const repositoryId = ref<number | null>(null);
-  const repository = computed(
-    () => repositoryId.value && Repository.findById(repositoryId.value),
-  );
+
+  const repository = computed(() => {
```

Comment:
this is mostly because of TS, previous version would return `0 | Repository | null | undefined` type, which would throw error in `CopyActivty` when attempting to get schema from the course (attempting to get schema of a 0).